src/Storefront/DependencyInjection/StorefrontMigrationReplacementCompilerPass.php

```diff
@@ -22,7 +22,11 @@ public function process(ContainerBuilder $container): void
         $migrationSourceV4->addMethodCall('addDirectory', [$migrationPath . '/V6_4', 'Shopware\Storefront\Migration\V6_4']);
         $migrationSourceV3->addMethodCall('addReplacementPattern', ['#^(Shopware\\\\Storefront\\\\Migration\\\\)V6_4\\\\([^\\\\]*)$#', '$1$2']);
 
-        $migrationSourceV5 = $container->getDefinition(MigrationSource::class . '.core.V6_5');
-        $migrationSourceV5->addMethodCall('addDirectory', [$migrationPath . '/V6_5', 'Shopware\Storefront\Migration\V6_5']);
+        $migrationPathV5 = $migrationPath . '/V6_5';
+        
+        if (@\is_dir($migrationPathV5)) {
```

Comment:
why ignore the error of is_dir?

Reply:
Habits. I learned to mute it.

Reply:
@shyim shall I un-mute?

Reply:
yep I am very sure that the mute doesnt work anymore with php 8

Reply:
I think in this case the "error" is still muted since it is actually a warning and not a fatal error, see:
https://www.php.net/manual/en/function.is-dir.php#refsect1-function.is-dir-errors

The change with php8 was only that fatal errors are muted:
https://www.php.net/manual/en/language.operators.errorcontrol.php

Nevertheless I am not sure when `\is_dir` is actually emitting a warning, so I am not sure if the warning should be silenced or not.

Reply:
I just thought about it:

php man for `is_dir` says

> Upon failure, an E_WARNING is emitted. 

Do we want that issue hinted? All of us don't expect an issue. So this is good without the mute for a working scenario. When the software behaves weird, you want to find out. Shopware will behave weird when migrations are missing! So when raising error output levels we would see a warning about a bad directory. As warnings can already be muted globally and it is presumable not a false positive we should unmute it.